riak_test/opbcounter_test.erl

```diff
@@ -33,7 +37,7 @@ confirm() ->
     decrement_test(Nodes),
     transfer_test(Nodes),
     conditional_write_test(Nodes),
-    rt:clean_cluster(Nodes),
+    %rt:clean_cluster(Nodes),
```

Comment:
I think we still need it - at the end of every independent test case. 
Tests must be independent, and cleaning up the test fixture is essential for this.
In the long run, someone will otherwise introduce "bugs" there.